crates/hir/src/semantics.rs

```diff
@@ -1702,6 +1702,10 @@ impl SemanticsScope<'_> {
             |name, id| cb(name, id.into()),
         )
     }
+
+    pub fn extern_crates(&self) -> Vec<Name> {
+        self.resolver.extern_crates_in_scope()
+    }
```

Comment:
Similar stuff here, but map from `ModuleId` to `Module`
```suggestion
    pub fn extern_crates(&self)  -> impl Iterator<Item = (Name, Module)> {
        self.resolver.extern_crates_in_scope().map(...)
    }
```